src/status_im2/contexts/wallet/events.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im2.contexts.wallet.events
+  (:require [taoensso.timbre :as log]
+            [re-frame.core :as re-frame]))
+
+(re-frame/reg-event-fx :wallet-2/get-wallet-token
+ (fn [_ [accounts]]
+   (let [params (map :address accounts)]
+     {:json-rpc/call [{:method     "wallet_getWalletToken"
+                       :params     [params]
+                       :on-success [:wallet-2/get-wallet-token-success]
```

Comment:
#(rf/dispatch

Reply:
I used `#(rf/dispatch` before but it was mentioned here https://github.com/status-im/status-mobile/pull/17447#discussion_r1340545397 to use this style

Reply:
i don't see anything about on-success function in the comment, yes i see that it doesn't have dispatch, probably by mistake or

Reply:
Done

Reply:
https://github.com/status-im/status-mobile/pull/17447#discussion_r1340547419 @flexsurfer found the comment